java/flight/flight-sql-jdbc-core/src/main/java/org/apache/arrow/driver/jdbc/ArrowFlightJdbcFlightStreamResultSet.java

```diff
@@ -45,15 +54,18 @@
  */
 public final class ArrowFlightJdbcFlightStreamResultSet
     extends ArrowFlightJdbcVectorSchemaRootResultSet {
-
+  private static final String BLOCKING_QUEUE_PARAM = "buffersize";
```

Comment:
Can we add this to the documentation?

Can we add a test of setting this parameter?